docker/pwsh-ews/Dockerfile

```diff
@@ -0,0 +1,6 @@
+
+FROM demisto/powershell:7.0.1.9103
+
+RUN pwsh -c "Set-PSRepository -Name PSGallery -InstallationPolicy Trusted -ErrorAction Stop"
+RUN pwsh -c "Install-Module -ErrorAction Stop -Scope AllUsers PartnerCenter -RequiredVersion 3.0.10 -Force"
+RUN pwsh -c "Install-Module -ErrorAction Stop -Scope AllUsers ExchangeOnlineManagement -RequiredVersion 2.0.4-Preview2 -AllowPrerelease -Force"
```

Comment:
Do we still need this? As we are now using the PartnerCenter integration?

Reply:
I don't need any package installed, Implemeted PartnerCenter as API call.